src/stac_api_validator/validations.py

```diff
@@ -297,6 +298,16 @@ def is_geojson_type(maybe_type: Optional[str]) -> bool:
     )
 
 
+def get_catalog(data_dict: Dict[str, Any], r_session: Session) -> Catalog:
+    stac_io = StacIO.default()
+    if r_session.headers and r_session.headers.get("Authorization"):
+        stac_io.headers = r_session.headers  # noqa, type: ignore
+        stac_io.headers["Accept-Encoding"] = "*"
```

Comment:
Is this line related to the authorization at all? Does it need to be set even if the authorization headers aren't set?

Reply:
I'm going to resolve this comment so we can continue to discuss in https://github.com/stac-utils/stac-api-validator/pull/316#discussion_r1179260335